### PR TITLE
Companion PR to NOAA-EMC/jcb/pull/7

### PR DESCRIPTION
**Pull request by danholdaway**
Companion PR: NOAA-EMC/jcb/pull/7

- Rename the atmosphere YAML files to be `atmosphere_<class>.yaml.j2`
- Change all the keys associated with the atmosphere to be `{{atmosphere_<key>}}`
- Add the marine YAML files.

### CODE REVIEW

**Review by CoryMartin-NOAA — APPROVED** [review]

**Review by DavidNew-NOAA — APPROVED** [review]

File: model/atmosphere/atmosphere_background_error_hybrid_gsibec_bump.yaml.j2 [hunk 1]
```diff
@@ -5,50 +5,50 @@ components:
...
-      gsi akbk: *akbk
-      gsi error covariance file: *gsiberr
-      gsi berror namelist file: *gsibnml
-      processor layout x direction: *layout_gsib_x
-      processor layout y direction: *layout_gsib_y
+      gsi akbk: {{atmosphere_fv3jedi_files_path}}/akbk.nc4
+      gsi error covariance file: {{atmosphere_gsibec_path}}/gsi-coeffs-gfs-global.nc4
+      gsi berror namelist file: {{atmosphere_gsibec_path}}/gfs_gsi_global.nml
+      processor layout x direction: {{atmosphere_layout_gsib_x}}
+      processor layout y direction: {{atmosphere_layout_gsib_x}}
```
Comment (RussTreadon-NOAA) [inline model/atmosphere/atmosphere_background_error_hybrid_gsibec_bump.yaml.j2]: Typo:  `atmosphere_layout_gsib_x` should be `atmosphere_layout_gsib_y` for `processor layout y direction`.

Comment (danholdaway) [inline model/atmosphere/atmosphere_background_error_hybrid_gsibec_bump.yaml.j2]:
```suggestion
      processor layout y direction: {{atmosphere_layout_gsib_y}}
```

Comment (danholdaway) [reply model/atmosphere/atmosphere_background_error_hybrid_gsibec_bump.yaml.j2]: Fixed. Thanks @RussTreadon-NOAA

**Review by RussTreadon-NOAA — APPROVED** [review]
Approve.

**Comment by RussTreadon-NOAA** [comment]
@danholdaway:  Can we merge this PR into `develop`?

**Comment by danholdaway** [comment]
@RussTreadon-NOAA  I'm not sure about merging because it will confuse what is being tested in g-w. If ultimately we need jcb and gdasapp on develop in order to test and merge https://github.com/NOAA-EMC/global-workflow/pull/2665 then merging this will then mean testing these changes in that PR. I would prefer to merge that PR (and associated jcb/gdas PRs) first. Unless I'm missing something if we merge these now then we may have to end up testing them with https://github.com/NOAA-EMC/global-workflow/pull/2665.

**Comment by RussTreadon-NOAA** [comment]
I'm fine with getting g-w [#2665](https://github.com/NOAA-EMC/global-workflow/pull/2665) across the line first.  g-w [#2665](https://github.com/NOAA-EMC/global-workflow/pull/2665) has been approved by  Cory, Guillaume, and me.  I don't know what else we can do to accelerate its merger into g-w `develop`

**Pull request merged**